src/opengl/renderer_ogl.cpp

```diff
@@ -17,11 +17,34 @@
 #include "opengl_utils.h"
 #include "utils/logger.h"
 
+#define BUFFER_OFFSET(i) (static_cast<uint8_t*>(nullptr) + i)
+
 namespace {
-    GLuint s_HARDCODED_PROGRAM_REMOVE_ME; // TODO: Remove
-    const uint32_t HARDCODED_VERTEX_ELEMENT_SIZE = sizeof(float) * 4; // TODO: Remove
+    GLuint s_HARDCODED_PROGRAM_REMOVE_ME;
```

Comment:
Not that it matters, but, why not keep the `// TODO: Remove` ?
easy to always search for `TODO` when cleaning.